craigslist-apartments.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+let axios = require('axios');
+let cheerio = require('cheerio');
+
+function mountURL(filter) {
+ let url = `https://vancouver.craigslist.org/search/apa?availabilityMode=0&bundleDuplicates=1&hasPic=1`;
+ if(filter.maxPrice) {
+   url = url + `&max_price=${filter.maxPrice}`;
+ }
+ if(filter.minPrice) {
+   url = url + `&min_price=${filter.minPrice}`; 
+ }
+ if(filter.kmFromPostalCode) {
+   url = url + `&search_distance=${filter.kmFromPostalCode}`; 
+ }
+ if(filter.postalCode) {
+   url = url + `&postal=${filter.postalCode}`; 
+ }
+ if(filter.minSqft) {
+   url = url + `&minSqft=${filter.minSqft}`; 
+ }
+ return url;
+}
+
+function getListOfRentals(filter) {
+  let mountedURL = mountURL(filter);
+  let listOfRentals = getListOfRentalsFromCraigslist (0, [], mountedURL); 
+  return listOfRentals;
+}
+
+function getListOfRentalsFromCraigslist(startIndex, listOfRentals, mountedURL) {
```

Comment:
nitpick: you could have called this `getCraigslistListings`

Reply:
Done.